server/src/main/java/org/jboss/as/server/mgmt/domain/ServerBootOperationsService.java

```diff
@@ -153,8 +154,9 @@ public void completed(final ModelNode result) {
         }
 
         @Override
-        public void failed(Exception e) {
-            setFailed(e);
+        public void failed(final Exception e) {
```

Comment:
Could you add a Java doc describing the argument cannot be null?

Reply:
Fixed.